### ISSUE TITLE
Mac edk.dll not found

### ISSUE THREAD

**Issue by mrsufgi** [Emotiv/community-sdk#128]
Hey, I'm working on a Unity project, I added the C# dll wrapper into the Unity project. (DotNetEmotivSDK-vs2010.dll). Added the edk.framework file from lib into Library\Framework
It does work on my Windows PC but unfortunately not on my mac, I really need to be able to run this on my Mac. 

anyone else experiencing that? or was able to run it on mac?  

got 'DllNotFoundException: edk.dll 
Emotiv.EdkDll.IEE_EmoEngineEventCreate();

**Comment by duytan90** [comment]
Hi @mrsufgi, you should try latest version of Emotiv Unity Plugin (currently it's v3.3.3.1). It should have the required edk.framework that you mentioned.

**Comment by mrsufgi** [comment]
I'll try that , thanks :)

**Comment by mrsufgi** [comment]
@duytan90 do you mean Emotiv Unity plugin - https://www.emotiv.com/product/unity3d-support-pack/ version 3.3.3.1?

**Comment by duytan90** [comment]
@mrsufgi please excuse me, yes I did mention the version on Emotiv Store that can run on other platforms. It seams not clear that the Community version can only run on Windows platform.

**Comment by nlhnt** [comment]
Hello, sorry for replying to such an old issue, but mine is really similar.
I am getting the exact same prompt while using Unity in Windows.
I put the .NET dll into assests, and made a directory for edk.dll (and other dlls from SDK) in C:\Emotiv\.
Then I created a system env. variable called "EMOTIV" that links to the previously mentioned directory.
What else should I do to link the edk.dll?
I've went through the documentation but found no help there, sorry if I missed anything.
